portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/RemoteStagingWithVersioning.testcase

```diff
@@ -43,95 +43,109 @@ definition {
 
-		Staging.gotoPublishToLive();
+			Staging.gotoPublishToLive();
 
-		Staging.configurePublishToLive(uncheckContentNameList = "Web Content");
+			Staging.configurePublishToLive(uncheckContentNameList = "Web Content");
 
-		Staging.publishToLive(remoteStaging = "true");
+			Staging.publishToLive(remoteStaging = "true");
+		}
 
-		User.logoutPG();
+		task ("Then: Verify the Page is published to live and Web Content is not published") {
+			User.logoutPG();
 
-		User.firstLoginUI(
-			password = "test",
-			specificURL = "http://localhost:9080",
-			userEmailAddress = "test@liferay.com");
+			User.firstLoginUI(
+				password = "test",
+				specificURL = "http://localhost:9080",
+				userEmailAddress = "test@liferay.com");
 
-		Navigator.gotoSitePage(
-			baseURL = "http://localhost:9080",
-			pageName = "Staging Test Page",
-			siteName = "Remote Site");
+			Navigator.gotoSitePage(
+				baseURL = "http://localhost:9080",
+				pageName = "Staging Test Page",
+				siteName = "Remote Site");
 
-		AssetPublisherPortlet.configureDynamicAssetSelectionPG(portletName = "Asset Publisher");
+			AssetPublisherPortlet.configureDynamicAssetSelectionPG(portletName = "Asset Publisher");
 
-		Refresh();
+			Refresh();
 
-		Alert.viewInfoMessageSpecific(infoMessage = "There are no results.");
+			Alert.viewInfoMessageSpecific(infoMessage = "There are no results.");
+		}
 	}
 
 	@priority = "3"
 	test PublishWCViaRemoteStaging {
 		property test.name.skip.portal.instance = "RemoteStagingWithVersioning#PublishWCViaRemoteStaging";
 
-		JSONLayout.addWidgetToPublicLayout(
-			groupName = "Site Name",
-			layoutName = "Staging Test Page",
-			widgetName = "Web Content Display");
+		task ("Given: Add Web Content Display on Page") {
+			JSONLayout.addWidgetToPublicLayout(
+				groupName = "Site Name",
+				layoutName = "Staging Test Page",
+				widgetName = "Web Content Display");
+		}
 
-		Navigator.gotoSitePage(
-			pageName = "Staging Test Page",
-			siteName = "Site Name");
+		task ("When: Add a new Web Content to Web Content Display and publish Page to Remote Live") {
+			Navigator.gotoSitePage(
+				pageName = "Staging Test Page",
+				siteName = "Site Name");
 
-		WebContentDisplayPortlet.add(
-			webContentContent = "Web Content Content",
-			webContentTitle = "Web Content Title");
+			WebContentDisplayPortlet.add(
+				webContentContent = "Web Content Content",
+				webContentTitle = "Web Content Title");
 
-		WebContentDisplayPortlet.viewContent(
-			webContentContent = "Web Content Content",
-			webContentTitle = "Web Content Title");
+			WebContentDisplayPortlet.viewContent(
+				webContentContent = "Web Content Content",
+				webContentTitle = "Web Content Title");
 
-		Staging.gotoPublishToLive();
+			Staging.gotoPublishToLive();
 
-		Staging.publishToLive(remoteStaging = "true");
+			Staging.publishToLive(remoteStaging = "true");
+		}
 
-		User.logoutPG();
+		task ("Then: Verify Web Content is published to Remote Live and can't edit") {
```

Comment:
Case sensitive:
Then: Verify web content entry is published to remote live and user cannot edit web content entry on remote site.